client/components/pie-chart/style.scss

```diff
@@ -0,0 +1,80 @@
+.pie-chart__chart-section-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__chart-section-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__chart-section-2 {
+	fill: $blue-light;
+}
+
+.pie-chart__legend-sample-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__legend-sample-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__legend-sample-2 {
+	fill: $blue-light;
+}
+
+.pie-chart__chart {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.pie-chart__legend-item {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+}
+
+.pie-chart__legend-item-title {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+.pie-chart__legend-item-sample {
+	padding-right: 4px;
+	align-content: center;
+}
+
+.pie-chart__title {
+	text-align: center;
+	font-size: 24px;
+}
+
+.pie-chart__legend-item-title-name {
+	font-size: 20px;
```

Comment:
I've just learned that recently, we shouldn't be using this font size. [Here](https://wordpress.com/design-handbook/typography/) are the allowed values.